Library/Homebrew/requirements/minimum_macos_requirement.rb

```diff
@@ -13,4 +13,8 @@ def initialize(tags)
   def message
     "OS X #{@version.pretty_name} or newer is required."
   end
+
+  def display_s
+    "MinimumMacOS = #{@version}"
```

Comment:
Done. Using `>=` instead of `>`, since that matches the version specified in the requirement.

```
Required: macOS >= 10.7 ✔
```